x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -215,7 +217,12 @@ export class HeadlessChromiumDriverFactory {
   }
 
   getPageExit(browser: Browser, page: Page) {
-    const pageError$ = Rx.fromEvent<Error>(page, 'error').pipe(
+    const pageErrorSubject$ = new Rx.Subject<Error>();
```

Comment:
nit: I think RX has a prescribed way to handle event emitter errors: https://www.learnrxjs.io/learn-rxjs/operators/creation/fromevent.